src/System.Runtime.Extensions/tests/System/IO/Path.cs

```diff
@@ -61,22 +95,50 @@ public static void GetExtension(string expected, string path)
     [Fact]
     public static void GetFileName()
     {
+        Assert.Equal(null, Path.GetFileName(null));
```

Comment:
> you could send it all through Path.Combine(params string[] paths)

It also leads to complications, since some of the inputs are not destined to be part of that path but instead the expected output.

> And there's always good old [MemberData].

Sure, and I considered that, but in a case like this it makes things harder to understand (at least for me) rather than cleaning things up, and the benefit of having each test case isolated for a small number of asserts like this is relatively minimal.

I'm going to leave this as-is.  I have converted a few of the other tests into theories, though.